datavault-broker/src/main/java/org/datavaultplatform/broker/controllers/PermissionsController.java

```diff
@@ -0,0 +1,125 @@
+package org.datavaultplatform.broker.controllers;
+
+import org.datavaultplatform.broker.services.PermissionsService;
+import org.datavaultplatform.common.model.PermissionModel;
+import org.datavaultplatform.common.model.RoleModel;
+import org.jsondoc.core.annotation.*;
+import org.jsondoc.core.pojo.ApiVerb;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/permissions")
+@Api(name="Permissions", description = "Interact with DataVault Roles and Permissions")
+public class PermissionsController {
+
+    private PermissionsService permissionsService;
+
+    public void setPermissionsService(PermissionsService permissionsService) {
+        this.permissionsService = permissionsService;
+    }
+
+    @ApiMethod(
+            path = "/permissions/role}",
+            verb = ApiVerb.POST,
+            description = "Create a new DataVault Role",
+            consumes = { MediaType.APPLICATION_JSON_VALUE },
+            produces = { MediaType.APPLICATION_JSON_VALUE },
+            responsestatuscode = "200 - OK"
+    )
+    @ApiHeaders(headers={
+            @ApiHeader(name="X-UserID", description="DataVault Broker User ID"),
+            @ApiHeader(name="X-Client-Key", description="DataVault API Client Key")
+    })
```

Comment:
I'm not sure that this is needed - it got copied over from an endpoint on the `UserController` which doesn't explicitly require these headers. I need to determine if these are required indirectly or whether I've copied over an incorrect piece of documentation